sample.btcusd.limit_only.config.js

```diff
@@ -46,14 +46,33 @@ module.exports = {
         // set to 0 or remove ENV var to have default behavior of canceling on the next
         // action timer
         CPBB_REBUY_CANCEL: 60 * 24 * 2,
-        // if there are twelve unfilled limit orders remaining on the books, expire them
+        // if there are this many unfilled limit orders remaining on the books, expire them
         // and rebuild the limit order set immediately using the sum total of funds
         // used for all the limit orders that existed, starting with the price at the highest limit value
         // using the rebuy config to create new orders
         // NOTE: if you use this setting, it is recommended that you set it higher than
         // the number of items in your CPBB_REBUY config so it doesn't excessively rebuild
         // the same orders over and over
-        CPBB_REBUY_REBUILD: 12,
+        CPBB_REBUY_REBUILD: 35,
+        // max $ spend on limit resells
+        CPBB_RESELL_MAX: 50,
+        // minimum order is .0001 BTC ($5 at $50K)
+        // rebuy logic will place up orders at this size until CPBB_REBUY_MAX is reached
+        CPBB_RESELL:
+          '.0001@4,.0002@5,.0003@6,.0004@7,.0005@8,.0006@9,.0007@10,.0008@11,.0009@12,.001@13,.0015@14,.002@15,.004@20,.008@25,.016@30,.032@35,.064@40,.128@45,.256@50,.512@55,1@60,2@65,4@70,8@75,16@80,32@85,64@90,128@95,200@96,200@97,200@98,200@99',
+        // when should we cancel limit orders?
+        // default behavior is on the next action point (if they didn't fill)
+        // if CPBB_REBUY_CANCEL is set, this is a number of minutes after the limit order
+        // creation timestamp that it will be considered ready to cancel if not filled
+        //  CPBB_RESELL_CANCEL: 60 * 24 * 14,
+        // if there are this many unfilled limit orders remaining on the books, expire them
+        // and rebuild the limit order set immediately using the sum total of funds
+        // used for all the limit orders and starting with the price at the highest limit value
+        // using the resell config to create new orders
+        // NOTE: if you use this setting, it is recommended that you set it higher than
+        // the number of items in your CPBB_RESELL config so it doesn't excessively rebuild
+        // the same orders over and over
+        CPBB_RESELL_REBUILD: 35,
```

Comment:
I think we should be careful with the resell configurations. This automation is meant to build a position, so the emphasis should be on accumulating with profits being taken at the APY threshold. When I test this, I will try it with just 1 resell configuration at 5% with an immediate cancel by the next run. This would allow the automation to take advantage of rapid short term volatility without compromising the long term goal.